src/components/_animated.less

```diff
@@ -25,3 +25,4 @@
 @import "animated/_loading-btn";
 @import "animated/_spring.less";
 @import "animated/_papercurl.less";
+@import "animated/vibrate.less";
```

Comment:
One more thanks for your PR!

Please start your include less file with an underscore: _vibrate.less. It helps Gulp distinguish the include files from the main file.